setup.py

```diff
@@ -20,7 +20,8 @@
 DEPENDENCIES = [
     "ujson",
```

Comment:
@smpiano @bitsofbits we have experienced some weird things with the version 2.0.0 of ujson, should we pin it to 1.35?

Reply:
@enriquetuya , @smpiano ujson isn't being used anywhere anymore, so I'll delete.